### DISCUSSION TITLE
RaspberryMatic on Synology VMM: No bootup messages?

### DISCUSSION THREAD

**Discussion by DcFamas** [OpenCCU/OpenCCU#1697] [🇺🇸 Q&A] [answered]
Hey there,
thanks for your awesome software!
I use raspberrymatic installed via. OVA on my Synology Virtual Machine Manager.
Because of many powerloss the raspberrymatic virtual machine won't start normally.
It always want to boot in "raspberrymatic recovery" after a reboot of the NAS. How is it possible to change the setting to the regular bootorder?

**Comment by jens-maus** [comment]
Sorry, but if you shutdown or reboot your Synology NAS with VMM running, you have to make sure to first shutdown the RaspberryMatic VM or you may risk to get an invalid RM installation afterwards. So you question here is more a general Synology VMM question and not really related to RaspberryMatic. In fact, RaspberryMatic normally (if everything is fine) automatically boots into its main system and the recovery system is just used if it identified an error during bootup or if it processed an update.

**Reply by DcFamas** [reply]
Hey @jens-maus,
same is when I connect via. ssh and type in "shutdown". In the VMM console I can see, that raspberrymatic shutdown the services. When I turn on the machine the virtual machine still wants to boot into recovery over and over again. 

I get this message:
![grafik](https://user-images.githubusercontent.com/35306967/153209028-78d12e49-1f33-4859-bdde-82e32c61ba43.png)

**Comment by jens-maus** [comment]
Then you didn't read the documentation detailed enough:

https://github.com/jens-maus/RaspberryMatic/wiki/Installation-Synology-VMM

Pay special attention for step 7 (Virtuelle Festplattengröße einstellen) which you probably missed/skipped. Because you have to resize the userfs disk **before** the first start of the VM on Synology.

And if the VM was working properly before and after your "many powerloss" it doesn't anymore, chances are high that your userfs was simply destroyed to some extend. Thus, regenerate the VM or restore it from a snapshot and then restore your latest backup.

**Comment by DcFamas** [comment]
I'm pretty sorry, I really missed this part. I'm going to reinstall the whole vm and use my backup.
Best regards and thank you! :)

**Comment by DcFamas** [comment]
Im sorry that I have to reopen.
I reinstalled the whole vm. After the backup my failure shows up again. Maybe its more about the graphics?

**Reply by jens-maus** [reply]
And what happens if you boot a complete fresh VM without having restored the backup. Does it work then?

**Reply by DcFamas** [reply]
Yes, then everything is fine.

**Reply by DcFamas** [reply]
VGA settings changed to vga, everthing works fine now. :)

**Comment by jens-maus** [comment, marked as answer]
Well, then you probably didn't follow the documentation on installing a RaspberryMatic VM in Synology VMM. Because otherwise you must have followed the installation guide here:

https://github.com/jens-maus/RaspberryMatic/wiki/Installation-Synology-VMM

Take special not on step 6 (Konfiguration kontrollieren) where for the graphics card setting **vmvga** is suggested to be used.

**Reply by DcFamas** [reply]
Yes, but this was my problem. When I changed it from vmvga to vga the failure disappeared.

**Reply by jens-maus** [reply]
That's strange. I just tested it myself again and checked that `vmvga` is selected in Synology VMM for the RaspberryMatic VM. And here it works. I can just imaging that it might be browser related if `vmvga` works or not. However, if switching to pure `vga` worked for you, that's also fine of course.
